lib/spack/spack/package.py

```diff
@@ -1935,40 +1932,18 @@ def dependency_activations(self):
         return (spec for spec in self.spec.traverse(root=False, deptype='run')
                 if spec.package.extends(self.extendee_spec))
 
-    def activate(self, extension, ignore_conflicts=False, **kwargs):
-        """Make extension package usable by linking all its files to a target
-        provided by the directory layout (depending if the user wants to
-        activate globally or in a specified file system view).
+    def activate(self, extension, view, **kwargs):
+        """
+        Add the extension to the specified view.
 
         Package authors can override this method to support other
```

Comment:
Is this doc-string still accurate? Don't we now want to implement the `view_file_conflicts` and siblings?

Reply:
I updated the docstring to clarify. In some cases users would still want to override this function. It makes sense to override this function for an extendable package if that package maintains its own state related to activated extendees, for example python's easy-install.pth file.